src/amo/components/SearchForm/index.js

```diff
@@ -199,8 +223,8 @@ export class SearchFormBase extends React.Component {
           onSuggestionsClearRequested={this.handleSuggestionsClearRequested}
           onSuggestionsFetchRequested={this.handleSuggestionsFetchRequested}
           onSuggestionSelected={this.handleSuggestionSelected}
+          ref={(ref) => { this.autocomplete = ref; }}
           renderSuggestion={this.renderSuggestion}
-          renderInputComponent={(props) => <SearchInput {...props} />}
```

Comment:
So I guess you can completely remove this `SearchInput` component if you don't use it anymore. It was only used there.

Reply:
Argm, shoot, I originally did and then I think the rebase messed it up. Thanks for catching that.

Reply:
Oh, no, looking it's still removed 👍